apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -7,46 +7,72 @@ import InstanceContext from '../../../contexts/InstanceContext';
 
 const TREE_PATH = '/qListObjectDef';
 
-export default function ListBoxSearch({ model, keyboard, dense = false, visible = true }) {
+export default function ListBoxSearch({ selections, model, keyboard, dense = false, visible = true }) {
   const { translator } = useContext(InstanceContext);
   const [value, setValue] = useState('');
-  const [isSearching, setIsSearching] = useState(false);
+  const [, setAppliedSearch] = useState(false);
```

Comment:
Whats the point of a state that is never used?

Reply:
Yeah, it looks funny I know. But if you look further down in the code you see how the state is read in the `set` callback. The reason for doing like this is that the state is otherwise not up do date with the latest `appliedSearch`.
Anyhow, I had to refactor this again and got rid of this flag altogether now.